### ISSUE TITLE
H3 Slider [ feature request ]

### ISSUE THREAD

**Issue by siraxe** [AkaneTendo25/musubi-tuner#118]
Haven't tested all the new features yet but curious if it wouldn't be too hard to add slider training in future using image/text pairs ?
From my vibe code testing it kinda worked but it's messy so I won't try to PR it 😆

**Comment by AkaneTendo25** [comment]
Shouldn't be too hard. I'll try to add it next week.

**Comment by AkaneTendo25** [comment]
Didn't test extensively, but looks like it working.

This was a text-mode "darkness" slider, so no image/video dataset was used. The complete training input was one semantic direction:

```toml
mode = "text"
target_modality = "video"
guidance_strength = 1.0
latent_frames = 2
latent_height = 12
latent_width = 20

[[targets]]
positive = "a very dark nighttime scene with deep shadows and low-key lighting"
negative = "a bright daytime scene with high-key lighting and clear illumination"
target_class = "a cinematic scene"
```

Training command:

```bash
accelerate launch --num_cpu_threads_per_process 1 minimax_h3_train_slider.py \
  --slider_config darkness.toml \
  --dit /path/to/minimax_h3_fl2va_bf16.safetensors \
  --h3_training_mode fl2va \
  --text_encoder /path/to/qwen3vl_32b_minimax_h3_bf16.safetensors \
  --text_encoder_quantization nvfp4 --h3_text_encoder_blocks_to_stream 50 \
  --network_module networks.lora_minimax_h3 --network_dim 8 --network_alpha 8 \
  --optimizer_type AdamW --learning_rate 1e-4 --max_train_steps 100 --seed 1234 \
  --output_dir output --output_name darkness \
  --sdpa --mixed_precision bf16 --save_precision bf16 --gradient_checkpointing \
  --h3_convrot_int8 --h3_adaln_rank 16 \
  --h3_base_preservation_loss_weight 0.02 \
  --h3_base_preservation_probability 0.25
```

Validation videos:

All five were generated with the BF16 DiT at 1344x768, 5 seconds, 20 steps, seed `4242`, and the same prompt "A woman walks slowly through a modern city street as the camera tracks alongside her.". Only the LoRA multiplier changed.

**Base, 0:**

https://github.com/user-attachments/assets/9d0a1d6e-854f-4ee3-8fd2-f9aff10938f5

**Slider, +1:**

https://github.com/user-attachments/assets/d37d03e6-13f2-46ef-92ac-32fa3ae8df8d

**Slider, -1:**

https://github.com/user-attachments/assets/1097222f-9e13-4a2e-87d2-9a4df5f1993f

**Slider, +2:**

https://github.com/user-attachments/assets/e4bf6398-efde-4100-a9d8-4924707ba768

**Slider, -2:**

https://github.com/user-attachments/assets/99f71cf3-a322-49e5-9afa-dd732e22e53e

**Comment by siraxe** [comment]
Tested text slider , working great , sunny vs foggy

https://github.com/user-attachments/assets/ca552538-3393-42b4-be3d-0306b3d683c2

**Comment by siraxe** [comment]
Been playing with this a bit , works great for speedup slider 
Needed a bit more frames for this I think , not 100% sure
latent_frames = 12
latent_height = 24
latent_width = 40

<table>
<tr>
<td width="50%">

https://github.com/user-attachments/assets/0ccf8cde-517f-4b41-ae68-065cb7ebf48e

</td>
<td width="50%">
  
https://github.com/user-attachments/assets/c22bc4ff-ad88-44ea-8fc7-503fb42274a8

</td>
</tr>
</table>
